src/widgetastic/browser.py

```diff
@@ -815,6 +822,65 @@ def get_current_location(self):
         # useful if it is necessary to recognize current frame
         return self.execute_script('return self.location.toString()')
 
+    @property
+    def current_window_handle(self):
+        """Returns the current window handle"""
+        window_handle = self.selenium.current_window_handle
+        self.logger.debug("current_window_handle -> %r", window_handle)
+        return window_handle
+
+    @property
+    def window_handles(self):
+        """Returns all available window handles"""
+        handles = self.selenium.window_handles
+        self.logger.debug("window_handles -> %r", handles)
+        return handles
+
+    def switch_to_window(self, window_handle):
+        """switches focus to the specified window
+
+        Args:
+            window_handle: The name or window handle
+        """
+        self.logger.debug("switch_to_window -> %r", window_handle)
+        self.selenium.switch_to.window(window_handle)
+
+    def new_window(self, url, focus=False):
```

Comment:
shouldn't we return another instance of wt.core wrapper for window handles ?

Reply:
@izapolsk If I understand this question correctly. For these simple methods; I think, we won't need (currently it returning simple windows id). but Yup, we need to return instance when we will move framework as `multi window handling framework` https://github.com/RedHatQE/widgetastic.core/issues/159